### PR TITLE
[TASK] Introduce shorter navigation titles

### PR DESCRIPTION
**Pull request by linawolf**
Wherever in the menu the context makes clear the broader topic

### CODE REVIEW

File: Documentation/CodingGuidelines/CglRest/Index.rst [hunk 1]
```diff
@@ -1,3 +1,4 @@
+:navigation-title: reSTructuredText
```
Comment (garvinhicking) [inline Documentation/CodingGuidelines/CglRest/Index.rst:1]:
```suggestion
:navigation-title: reStructuredText
```

File: Documentation/CodingGuidelines/CodingBestPractices/Index.rst [hunk 2]
```diff
@@ -1,3 +1,4 @@
+:navigation-title: Best
```
Comment (garvinhicking) [inline Documentation/CodingGuidelines/CodingBestPractices/Index.rst:1]:
```suggestion
:navigation-title: PHP
```

File: Documentation/ApiOverview/CropVariants/Index.rst [hunk 3]
```diff
@@ -1,3 +1,4 @@
+:navigation-title: Cropping Images
```
Comment (garvinhicking) [inline Documentation/ApiOverview/CropVariants/Index.rst:1]:
```suggestion
:navigation-title: Cropping images
```

**Review by garvinhicking — DISMISSED** [review]

File: Documentation/Configuration/Typo3Information.rst [hunk 4]
```diff
@@ -1,3 +1,4 @@
+:navigation-title: Typo3Information
```
Comment (sarahmccarthy123) [inline Documentation/Configuration/Typo3Information.rst:1]:
```suggestion
:navigation-title: Typo3 Information
```

File: Documentation/CodingGuidelines/CodingBestPractices/Index.rst [hunk 5]
```diff
@@ -1,3 +1,4 @@
+:navigation-title: PHP
```
Comment (sarahmccarthy123) [inline Documentation/CodingGuidelines/CodingBestPractices/Index.rst:1]:
```suggestion
:navigation-title: PHP Best Practice
```

**Review by sarahmccarthy123 — COMMENTED** [review]
Added a suggestion for PHP titles otherwise there are two called "PHP" which might be a bit confusing

**Review by sarahmccarthy123 — APPROVED** [review]

**Pull request merged**